docs/specifications/tests/Nameserver-TP/nameserver14.md

```diff
@@ -0,0 +1,76 @@
+# nameserver14: Test for unknown version with unknown OPTION-CODE
+
+## test case identifier
+
+**NAMESERVER14** 
+
+### Objective
+
+This test case actually combines the test options in nameserver 10 and
+nameserver 11. 
+
+### Inputs
+
+"Child Zone" - The domain name to be tested.
+
+### Ordered description of steps to be taken to execute the test case
+
+1. Create an SOA query for the *Child Zone* with an OPT record with 
+   EDNS version set to "1" and  with an OPT record with
+   EDNS OPTION-CODE set to anything other than it is already assigned as in the
+[IANA-DNSSYSTEM-PARAMETERS] and no other EDNS options or flags.
```

Comment:
I have a problem with the wording since it sounds as if more than one OPT record is added. I suggest:
```
1. Create an SOA query for the *Child Zone* with an OPT record with 
   EDNS version set to "1" and  with EDNS OPTION-CODE set to an
   unassigned value (see [IANA-DNSSYSTEM-PARAMETERS]) and
   with no other EDNS options or flags.
```